6 completely rework readme (#11)
Danilado/jaba-is-you: README.md

> # The final project for the "Industrial Programming" course of [MSHP](https://informatics.ru)

Reviewer: I would make a separate Russian translation of the readme and move this disclaimer there. Nobody outside Moscow and its districts needs this disclaimer. Mention that this is a training project is already written below.

# The original game is called [Baba is You](https://store.steampowered.com/app/736260/Baba_Is_You/) 

> We don't take any credit on creating the game idea and /or mechanics. This is a project created for educational purposes with consent of original game's creator (Hempuli) through an email

Reviewer: ```suggestion
We don't take any credit for creating the game's idea, mechanics, sprites, and/or music. This is a project created for educational purposes with the consent of the original game's creator ([Hempuli](https://www.hempuli.com/)) via email.
```
